_test/test_mpi_wrappers/test_cluster_wrapper.m

```diff
@@ -366,24 +370,25 @@ function test_set_user_env_if_matlabpath_set(obj)
             [cluster,~,clOb] = obj.get_cluster_wrapper_tester(3);
             path = cluster.common_env_var('MATLABPATH');
             pathes = split(path,pathsep);
-            assertTrue(ismember(necessary_path,pathes));
-            assertFalse(ismember(test_dir,pathes));
+            assertFalse(obj.path_belongs(test_dir,pathes));
+            assertTrue(obj.path_belongs(necessary_path,pathes));
```

Comment:
please can we do the correct spelling of `paths` as otherwise someone may think it means something else

also it is perverse to switch the replacement lines around for old 369,370; much easier to review if true<->true, false<->false